modules/aspect_bazel_lib/metadata.json

```diff
@@ -1,19 +1,14 @@
 {
-    "homepage": "https://docs.aspect.build/rules/aspect_bazel_lib",
+    "homepage": "https://docs.aspect.dev/bazel-lib",
     "maintainers": [
         {
-            "name": "Alex Eagle",
-            "email": "alex@aspect.dev",
-            "github": "alexeagle"
-        },
-        {
-            "name": "Derek Cormier",
-            "email": "derek@aspect.dev",
-            "github": "kormide"
+            "email": "arijit.sen@machanirobotics.com",
+            "github": "arijit-sen",
+            "name": "Aspect team"
         }
     ],
     "repository": [
-        "github:aspect-build/bazel-lib"
+        "github:arijit-sen/bazel-lib"
```

Comment:
@meteorcloudy I was hoping that someone would try this: We need to insert a check that requires a manual review if this property fails.

Reply:
You mean we need a manual review whenever this field is changed, right?

Reply:
I'll work on this ASAP.